.github/workflows/kube-stack-test.yaml

```diff
@@ -28,5 +28,7 @@ jobs:
           kubectl wait --timeout=5m --for=condition=available deployment cert-manager -n cert-manager
           kubectl wait --timeout=5m --for=condition=available deployment cert-manager-webhook -n cert-manager
 
-      - name: Run chart-testing (install)
-        run: ct install --charts charts/opentelemetry-kube-stack
+      - name: Run Helm Install
+        run: |
+          helm dep build charts/opentelemetry-kube-stack
+          helm install opentelemetry-kube-stack charts/opentelemetry-kube-stack
```

Comment:
This is fine for now, but once we add a `ci` folder we'll want to go back to `ct install`.

Reply:
Ya we'll have to solve that later somehow